llama_index/llm_predictor/base.py

```diff
@@ -26,6 +26,9 @@
 
 logger = logging.getLogger(__name__)
 
+_COHERE_MAX_INPUT_SIZE = 2048
+_AI21_J2_MAX_INPUT_SIZE = 8192
```

Comment:
Can we add these to llama_index.constants as well, just so the context window definitions are consistent 

Could rename them something along the lines of `XX_CONTEXT_WINDOW` as well

Reply:
Ah, that's a good idea. Will do that right now :)